cmd/rpcdaemon/commands/root.go

```diff
@@ -0,0 +1,104 @@
+package commands
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"runtime"
+	"runtime/pprof"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ledgerwatch/turbo-geth/log"
+	"github.com/ledgerwatch/turbo-geth/node"
+)
+
+type Config struct {
+	remoteDbAdddress string
+	rpcListenAddress string
+	rpcPort          int
+	rpcCORSDomain    string
+	rpcVirtualHost   string
+	rpcAPI           string
+}
+
+var (
+	cpuprofile     string
+	cpuProfileFile io.WriteCloser
+
+	memprofile string
+	cfg        Config
+)
+
+func init() {
+	rootCmd.PersistentFlags().StringVar(&cpuprofile, "cpuprofile", "", "write cpu profile `file`")
+	rootCmd.PersistentFlags().StringVar(&cpuprofile, "memprofile", "", "write memory profile `file`")
```

Comment:
Using the wrong target variable `&cpuprofile`

Reply:
Well spotted!